pipeline/ingest/tracking.py

```diff
@@ -148,8 +151,7 @@ def make(self, key):
                 tfull = list(tpath.glob(tfile))
 
                 if not tfull or len(tfull) > 1:
-                    log.info('file mismatch: file: {} trial: {} ({})'.format(
-                        t, tmap[t], tfull))
+                    # log.info('file mismatch: file: {} trial: {} ({})'.format(t, tmap[t], tfull))
```

Comment:
not clear on commenting out - if too verbose, maybe adjust to log.debug?

Reply:
oh, mistake on my part, meant to uncomment, but yes let's change to debug, too verbose